ocp_utilities/operators.py

```diff
@@ -221,12 +221,13 @@ def install_operator(
             if not ns.exists:
                 ns.deploy(wait=True)
 
-        OperatorGroup(
-            client=admin_client,
-            name=name,
-            namespace=operator_namespace,
-            target_namespaces=target_namespaces,
-        ).deploy(wait=True)
+        if operator_namespace != "openshift-operators":
```

Comment:
why only `openshift-operators`? check if the resource exists in the namespace and if not create it

Reply:
When ns is `openshift-operators`, OperatorGroup `global-operators` already exists which is used by the operators.

Otherwise we are creating OperatorGroup from operator `name` which is not required in this case.

Reply:
this is not what I requested; if `OperatorGroup` exists do not create it

Reply:
updated the code to add this check.